docs/user/guides/management/index.rst

```diff
@@ -4,9 +4,13 @@ How-to guides: account management
 ⏩️ :doc:`Managing your Read the Docs for Business subscription </commercial/subscriptions>`
     Solving the most common tasks for managing Read the Docs subscriptions.
 
+🔒 :doc:`/guides/management/2fa`
+    Secure your account with two-factor authentication.
```

Comment:
We should probably have an overview of auth stuff in the docs as well. Just having a guide feels like a good starting point, but I think we want to mention it in other places as well. If there isn't an obvious place to list this as a feature, we should probably create one.

Reply:
yeah, I was having trouble finding a place where to put this content.